src/protocols/memory/ddr3.stanza

```diff
@@ -192,25 +191,8 @@ with:
 
 doc: \<DOC>
 Constructor for the DDR3 Data Link Constraint
-@param width Width of the DDR3 interface (enumerated type)
-@param diff-dqs-rs Differential Routing Structure constraints for DQS signals
-@param se-dq-rs Routing Structure constraints for DQ signals
 <DOC>
-public defn DDR3-Data-ACC-Constraint (
```

Comment:
Ya, removing these is a good idea, the `keyword-constructor=true` is a better approach to rely on.